pkg/component/watch_test.go

```diff
@@ -80,22 +79,17 @@ var StartChan = make(chan bool)
 // Mock PushLocal to collect changed files and compare against expected changed files
 func mockPushLocal(client *occlient.Client, componentName string, applicationName string, path string, out io.Writer, files []string, delFiles []string, isPushForce bool, globExps []string, show bool) error {
 	for _, gotChangedFile := range files {
-		found := false
 		// Verify every file in expected file changes to be actually observed to be changed
 		// If found exactly same or different, return from PushLocal and signal exit for watch so that the watch terminates gracefully
 		for _, expChangedFile := range ExpectedChangedFiles {
 			wantedFileDetail := CompDirStructure[expChangedFile]
 			if filepath.Join(wantedFileDetail.FileParent, wantedFileDetail.FilePath) == gotChangedFile {
-				found = true
 				ExtChan <- true
 				return nil
 			}
 		}
-		if !found {
-			ExtChan <- true
-			return fmt.Errorf("received %+v which is not same as expected list %+v", files, ExpectedChangedFiles)
-		}
 	}
+	ExtChan <- true
```

Comment:
@kadel Please have a look at this part.

Reply:
I'm not sure about this at all :-(

Reply:
After debugging I found that `found = true` was never called. `found` always stayed false and thus I removed the variable completely.
```
ineffectual assignment to `found` (ineffassign)
```
This is the error reported by golangci-lint. Also if if `found=true` does get executed, we are returning from the function after that, so changing the found value is useless.

Reply:
that makes sense